### PR TITLE
✨ Implement backend

### PR DESCRIPTION
**Pull request by MaxMuecket**
### Add dependencies:

- "mongodb": "^3.6.9"
- "http-proxy-middleware": "^2.0.0"
- "@types/mongodb": "^3.6.18"

### MongoDb

- Add MongoDb
- Connect MongoDB

### Routes

- `POST` plant
- `GET` plant
- `DELETE` plant

### CODE REVIEW

File: .storybook/middleware.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+const { createProxyMiddleware } = require('http-proxy-middleware');
```
Comment (lmachens) [inline .storybook/middleware.js:1]: Don't forget to load environments from your dotenv.

```ts
const dotenv = require('dotenv');
dotenv.config();
```

File: samples.http [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+Content-Type: application/json
+
+{"imagePath": "URL/....",
+"name": "Fern",
+"firstMonth": 0,
+"lastMonth": 11,
+"place": 2,
+"note": "TriTraTrulala",
+"active": false
+}
```
Comment (lmachens) [inline samples.http:15]: Not an issue, but the format looks weird

**Review by lmachens — COMMENTED** [review]
I like

File: .storybook/middleware.js (hunk 1 above)
Comment (MaxMuecket) [reply .storybook/middleware.js:1]: Thanks!

**Pull request merged**